<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newGameView</class>
 <widget class="QWidget" name="newGameView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #8A8557;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Battle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hostGameButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
border-color: #111111;
    

</string>
   </property>
   <property name="text">
    <string>Spiel leiten</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="joinGameButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
border-color: #111111;

</string>
   </property>
   <property name="text">
    <string>Spiel beitreten</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showMainMenuButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
border-color: #111111;
   
</string>
   </property>
   <property name="text">
    <string>Zurück zum Hauptmenü</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>780</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Abmelden vom Dienst</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
